pkg/apis/migration/migration.go

```diff
@@ -164,26 +171,26 @@ func GetVolumeMigrationService(ctx context.Context, volumeManager *cnsvolume.Man
 	return volumeMigrationInstance, nil
 }
 
-// GetVolumeID returns VolumeID for given VolumePath
+// GetVolumeID returns VolumeID for given VolumeSpec
 // Returns an error if not able to retrieve VolumeID.
-func (volumeMigration *volumeMigration) GetVolumeID(ctx context.Context, volumePath string) (string, error) {
+func (volumeMigration *volumeMigration) GetVolumeID(ctx context.Context, volumeSpec VolumeSpec) (string, error) {
 	log := logger.GetLogger(ctx)
-	info, found := volumeMigration.volumePathToVolumeID.Load(volumePath)
+	info, found := volumeMigration.volumePathToVolumeID.Load(volumeSpec.VolumePath)
 	if found {
-		log.Infof("VolumeID: %q found from the cache for VolumePath: %q", info.(string), volumePath)
+		log.Infof("VolumeID: %q found from the cache for VolumePath: %q", info.(string), volumeSpec.VolumePath)
 		return info.(string), nil
 	}
-	log.Infof("Could not retrieve VolumeID from cache for Volume Path: %q. volume may not be registered. Registering Volume with CNS", volumePath)
-	volumeID, err := volumeMigration.registerVolume(ctx, volumePath)
+	log.Infof("Could not retrieve VolumeID from cache for Volume Path: %q. volume may not be registered. Registering Volume with CNS", volumeSpec.VolumePath)
+	volumeID, err := volumeMigration.registerVolume(ctx, volumeSpec)
 	if err != nil {
-		log.Errorf("failed to register volume for VolumePath: %q, with err: %v", volumePath, err)
+		log.Errorf("failed to register volume for volumeSpec: %v, with err: %v", volumeSpec, err)
 		return "", err
 	}
-	log.Infof("successfully registered volume: %q with CNS. VolumeID: %q", volumePath, volumeID)
+	log.Infof("successfully registered volume: %q with CNS. volumeSpec: %v", volumeSpec, volumeID)
```

Comment:
Shouldn't this be: "successfully registered volumeSpec: %v with CNS. VolumeID: %q"

Reply:
yes.

changed to

```
log.Infof("successfully registered volumeSpec: %v with CNS. VolumeID: %v", volumeSpec, volumeID)
```